polaris/ocean/convergence/spherical/forward.py

```diff
@@ -0,0 +1,138 @@
+from polaris.ocean.model import OceanModelStep, get_time_interval_string
+
+
+class SphericalConvergenceForward(OceanModelStep):
+    """
+    A step for performing forward ocean component runs as part of a spherical
+    convergence test
+
+    Attributes
+    ----------
+    resolution : float
+        The resolution of the (uniform) mesh in km
+
+    package : Package
+        The package name or module object that contains the YAML file
+
+    yaml_filename : str
+        A YAML file that is a Jinja2 template with model config options
+
+    """
+
+    def __init__(self, component, name, subdir, resolution, base_mesh, init,
+                 package, yaml_filename='forward.yaml',
+                 output_filename='output.nc', validate_vars=None):
```

Comment:
One more thing that came up in `sphere_transport`: can we add an optional argument `options` (or `namelist_options`) and below `self.add_model_config_options(options=options)`?

Reply:
Yep, that makes sense to do.  Good idea!

Reply:
This has been added.